thesis-gui/src/components/HomePage.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect } from "react";
+import {
+    IonCard, IonCardContent, IonCardSubtitle, IonCardTitle,
+    IonContent,
+    IonLoading,
+    IonPage,
+    IonRouterLink, IonText
+} from "@ionic/react";
+import {useHistory} from "react-router";
+import {useCategoryContext} from "../context/CategoryContext";
+import {PageHeader} from "./common/PageHeader";
+
+const PAGE_NAME = 'Play';
+
+export const HomePage: React.FC = () => {
+    const { loadingState } = useCategoryContext();
+    const history = useHistory();
+
+    useEffect(() => {
```

Comment:
I don't think this `useEffect `is necessary here. It should be only in `CategorySelect` to redirect the user to error page, because we cannot display categories list if the request failed. In this component categories aren't necessarily required, because the user might be interested only in private quizes, so there is no reason to display error-page.

Reply:
True, it must have been left after copy-paste 🫣
Done